dev/sparktestsupport/modules.py

```diff
@@ -715,8 +715,13 @@ def __hash__(self):
         # unittests
         "pyspark.pandas.tests.indexes.test_base",
         "pyspark.pandas.tests.indexes.test_datetime",
-        "pyspark.pandas.tests.test_dataframe",
-        "pyspark.pandas.tests.test_dataframe_slow",
+        # "splits of pyspark.pandas.tests.test_dataframe",
+        "pyspark.pandas.tests.test_dataframe_splits.split_a_b",
```

Comment:
I think it'd be great if there's any good reference for splitting the test cases.

At least, I believe the file of tests should start with `test_` prefix in general.

Reply:
got it, will fix